docs/sources/tempo/metrics-generator/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 aliases:
-- /docs/tempo/latest/server_side_metrics/
-- /docs/tempo/latest/metrics-generator/
+  - ./server_side_metrics # /docs/tempo/<TEMPO_VERSION>/server_side_metrics/
+  - /docs/tempo/latest/metrics-generator/
```

Comment:
Looks like this alias still needs fixing. 

Since we're gonna want to remove the use of absolute aliases across all versions, perhaps it's best as a separate PR?

Reply:
@jdbaldry That would probably be good. There are a lot of  URLs that need to be brought up to date with the recommendations from the Writers' Toolkit. Do you have tooling that can do that?

Reply:
```suggestion
  - /docs/tempo/<TEMPO_VERSION>/metrics-generator/
```

Reply:
To clarify, I think the `/docs/tempo/latest/metrics-generator/` alias needs to be a relative alias because this is versioned documentation https://grafana.com/docs/writers-toolkit/write/front-matter/#versioned-projects